app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -787,18 +792,6 @@ public void onResume() {
                 if (!mIconUnsaved && !croppedIcon() && !mIconRemoved) {
                     setCardImage(thumbnail, Utils.retrieveCardImage(this, tempLoyaltyCard.id, ImageLocationType.icon), false);
                 }
-            } else {
-                setTitle(R.string.addCardTitle);
-            }
-
-            if (mFrontImageUnsaved && !croppedFrontImage()) {
-                setCardImage(cardImageFront, Utils.loadTempImage(this, TEMP_UNSAVED_FRONT_IMAGE_NAME), true);
-            }
-            if (mBackImageUnsaved && !croppedBackImage()) {
-                setCardImage(cardImageBack, Utils.loadTempImage(this, TEMP_UNSAVED_BACK_IMAGE_NAME), true);
-            }
-            if (mIconUnsaved && !croppedIcon()) {
-                setCardImage(thumbnail, Utils.loadTempImage(this, TEMP_UNSAVED_ICON_NAME), false);
             }
```

Comment:
These `setCardImage` calls should still stay

Reply:
Ok now i'm pretty positive everything is alright sorry for the amount of commits...

Reply:
Let'ss goooo hahahahha glad to hear that thanks for everything